android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/ScopedFirebaseCoreService.java

```diff
@@ -86,14 +85,14 @@ public ScopedFirebaseCoreService(Context context, RawManifest manifest, Experien
     updateFirebaseApp(mAppOptions, mAppName);
   }
 
-  private static String getEncodedExperienceId(ExperienceId experienceId) {
+  private static String getEncodedExperienceId(ExperienceKey experienceKey) {
     try {
-      String encodedUrl = experienceId.getUrlEncoded();
+      String encodedUrl = experienceKey.getUrlEncodedStableLegacyId();
```

Comment:
Thinking more about this, this needs to be the scope key because modern manifests won't necessarily have a legacy ID (full name).

We should keep in mind that we want our scope keys to be nicely URL-encodable (ideally URL-safe to begin with).